### ISSUE TITLE
"use a location descriptor instead" in Link tag

### ISSUE THREAD

**Issue by stigmat4j** [rackt/react-router#2677]
My link - 
```js
<Link to={`/account/metrics/spreads`} activeClassName="active" className="list-group-item"><span className="icon-svg spreads"></span>Spreads</Link>

```

And i see a warning in console
```
Warning: the query argument to createHref is deprecated; use a location descriptor instead
```

What should i change in Link tag?

**Comment by timdorr** [comment]
At the moment, you can solve this by locking your history version like so: `npm install --save history@1.31.1`. We will be releasing a version that fixes this warning soon.

**Comment by growlsworth** [comment]
Thanks for the update. Also version typo fix: `npm install --save history@1.13.1`

**Comment by danielfalk** [comment]
Do you mean 1.13.1?  The versions for history in npm only seem to go up to 1.15.0

**Comment by timdorr** [comment]
@danielfalk Yes, that is the correct version to lock to. 1.14.0 and 1.15.0 have depreciation warnings that Link currently hits, causing errors that aren't your fault. There aren't any bug fixes in these versions, so sticking to 1.13.1 is fine for now. We're prepping for a 2.0 release of history soon.
